src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
+// const deer = new Herbivore('Bembi');
+// const panther = new Carnivore('Bagira');
+// const lion = new Carnivore('King');
+// const rabbit = new Herbivore('Max');
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  name = this.name;
+  health = this.health;
+  hidden = false;
+
+  hide() {
+    if (this.hidden === false) {
+      this.hidden = true;
+    } else {
+      this.hidden = false;
+    }
```

Comment:
[FUNCTION NAMING] - The `hide` method should only set the `hidden` property to `true` and should not toggle its value. Please, revise the method to ensure it only hides the herbivore.